apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useCaseBlocksLogic/utils/getTabsToBlocksMap.tsx

```diff
@@ -0,0 +1,144 @@
+        />
+      ),
+    });
+  });
+
+  return blocks.build();
+};
+
+const createAssociatedCompanyDocumentBlocks = (
+  workflow: TWorkflowById,
+  { onReuploadNeeded, isLoadingReuploadNeeded }: TCaseBlocksCreationProps,
+) => {
+  const blocks = createBlocksTyped().addBlock();
+
+  const childWorkflows = workflow?.childWorkflows?.filter(
+    childWorkflow => childWorkflow?.context?.entity?.type === 'business',
+  );
+
+  if (!childWorkflows?.length) return [];
+
+  childWorkflows.forEach(childWorkflow => {
+    blocks.addCell({
+      type: 'nodeBlock',
+      value: (
+        <ChildDocumentBlocks
+          parentWorkflowId={workflow.id}
+          childWorkflow={childWorkflow}
+          parentMachine={workflow?.context?.parentMachine}
+          key={childWorkflow?.id}
+          onReuploadNeeded={onReuploadNeeded}
+          isLoadingReuploadNeeded={isLoadingReuploadNeeded}
+        />
+      ),
+    });
+  });
+
+  return blocks.build();
+};
+
+export const getTabsToBlocksMap = (
+  blocks: any[],
+  blocksCreationParams: TCaseBlocksCreationProps,
+  theme?: WorkflowDefinitionConfigTheme,
+) => {
+  const { workflow } = blocksCreationParams;
+
+  const [
+    websiteMonitoringBlock,
+    entityInfoBlock,
+    registryInfoBlock,
+    kybRegistryInfoBlock,
+    companySanctionsBlock,
+    ubosUserProvidedBlock,
+    ubosRegistryProvidedBlock,
+    directorsUserProvidedBlock,
+    directorsRegistryProvidedBlock,
+    directorsDocumentsBlocks,
+    storeInfoBlock,
+    websiteBasicRequirementBlock,
+    bankingDetailsBlock,
+    processingDetailsBlock,
+    mainContactBlock,
+    mainRepresentativeBlock,
+    mapBlock,
+    parentDocumentBlocks,
+    associatedCompaniesBlock,
+    associatedCompaniesInformationBlock,
+    processTrackerBlock,
+    websiteMonitoringBlocks,
+    documentReviewBlocks,
+    businessInformationBlocks,
+  ] = blocks;
+
+  const defaultTabsMap = {
+    summary: [...processTrackerBlock, ...websiteMonitoringBlock, ...entityInfoBlock],
+    company_information: [
+      ...registryInfoBlock,
+      ...kybRegistryInfoBlock,
+      ...companySanctionsBlock,
+      ...bankingDetailsBlock,
+    ],
+    store_info: [...storeInfoBlock, ...processingDetailsBlock, ...websiteBasicRequirementBlock],
+    documents: [...parentDocumentBlocks, ...directorsDocumentsBlocks],
+    ubos: [
+      ...ubosUserProvidedBlock,
+      ...ubosRegistryProvidedBlock,
+      ...(createKycBlocks(workflow as TWorkflowById) || []),
+    ],
+    associated_companies: [
+      ...associatedCompaniesBlock,
+      ...associatedCompaniesInformationBlock,
+      ...createKycBlocks(workflow as TWorkflowById),
+      ...createAssociatedCompanyDocumentBlocks(workflow, blocksCreationParams),
+    ],
+    directors: [
+      ...directorsUserProvidedBlock,
+      ...directorsRegistryProvidedBlock,
+      ...(createKycBlocks(workflow as TWorkflowById) || []),
+    ],
+    website_monitoring: [...websiteMonitoringBlocks],
+  };
```

Comment:
Ensure maps use `as const`.
```suggestion
  } as const;
```